### PR TITLE
Adding fixes for sso and making sso by default for all the jaggery apps

### PR DESCRIPTION
**Pull request by Megala21**

### CODE REVIEW

File: modules/core/distribution/src/assembly/filter.properties [hunk 1]
```diff
@@ -25,3 +25,5 @@ carbon.version=4.4.9
 default.server.role=IoTServer
 cep.server.role=ComplexEventProcessor
 das.server.role=DataAnalyticsServer
+emm.analytics.role=EMMAnalytics
```
Comment (charithag) [inline modules/core/distribution/src/assembly/filter.properties]: IMO this property should be iot.analytics.role=IoTAnalytics. Please update the usages also.

File: modules/core/distribution/src/repository/conf/carbon.xml [hunk 2]
```diff
@@ -79,6 +79,8 @@
 		<Role>${default.server.role}</Role>
 		<Role>${das.server.role}</Role>
 		<Role>${cep.server.role}</Role>
+		<Role>${emm.analytics.role}</Role>
```
Comment (charithag) [inline modules/core/distribution/src/repository/conf/carbon.xml]: IMO this property should be iot.analytics.role

**Review by charithag — COMMENTED** [review]
Please kindly do requested changes

File: modules/core/distribution/src/assembly/filter.properties (hunk 1 above)
Comment (Megala21) [reply modules/core/distribution/src/assembly/filter.properties]: We are using IoTServer role in gadgets and dashboards, hence remove this with 090da7941920d7cd57a473b3abe894d4901eb095

File: modules/core/distribution/src/repository/conf/carbon.xml (hunk 2 above)
Comment (Megala21) [reply modules/core/distribution/src/repository/conf/carbon.xml]: We are using IoTServer role in gadgets and dashboards, hence remove this with 090da7941920d7cd57a473b3abe894d4901eb095

**Pull request merged**